### PR TITLE
Add release tooling

### PR DESCRIPTION
**Pull request by ajeddeloh**
Add release tagging/building scripts as well as a checklist.

### CODE REVIEW

File: build_releases [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+build_release() {
+    export NAME="fcct-${1}"
+    echo "building ${NAME}"
+    go build -o ${BIN_PATH}/${NAME} internal/main.go
+}
+
+export GOOS=linux
+export GOARCH=amd64
+build_release x86_64-unknown-linux-gnu
```
Comment (jlebon) [inline build_releases]: Minor/optional: could just make `build_release` take the OS and arch as paramters.

File: build_releases [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -eu
```
Comment (jlebon) [inline build_releases]: The bash scripter in me is itching to add a `o pipefail` to that.

File: tag_release.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+[ $# == 2 ] || { echo "usage: $0 version commit" && exit 1; }
```
Comment (jlebon) [inline tag_release.sh]: `<version> <commit>` ?

File: build_releases (hunk 1 above)
Comment (ajeddeloh) [reply build_releases]: eh, I'll leave it off until it gets hairy. This matches the Ignition one too.

**Comment by ajeddeloh** [comment]
fixed.

**Pull request merged**